contracts/src/v0.8/functions/dev/v1_X/FunctionsBilling.sol

```diff
@@ -129,6 +131,19 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
     return SafeCast.toUint96((1e18 * gasPriceWei) / getWeiPerUnitLink());
   }
 
+  function _getL1FeeJuels() private view returns (uint96) {
+    // Will return non-zero on chains that have this enabled
+    uint256 l1CostWei = ChainSpecificUtil._getCurrentTxL1GasFees(msg.data);
+
+    if (l1CostWei == 0) {
```

Comment:
Remove this maybe? It's unlikely it will ever happen and the logic below still works with 0

Reply:
It's very likely that will happen (happens on every non L2 network)

Reply:
Right, thanks! Then yeah, keep it.

Reply:
Here are the gas savings from this check
![image](https://github.com/smartcontractkit/chainlink/assets/28818476/6b0716a1-3ee4-472f-a653-f655c964f006)